tools/gdb/README.md

```diff
@@ -0,0 +1,12 @@
+# Custom gdb scripts
+Custom gdb scripts for gdb debugging and autoloading of these scripts.
+individual scripts are located in the `gdb_scripts/` and the top level directory is reserved for loader scripts `load_gdb_scripts.py`.
+
+### Usage
+To loads scripts from within gdb call `source /path/to/load_gdb_scripts.py`.
+If you have compiled WiredTiger with the `-DHAVE_SHARED=1` flag then these scripts 
```

Comment:
Is it ENABLE_SHARED or HAVE_SHARED? The cmake file checks against ENABLE_SHARED.

Reply:
Good catch! I that's a typo on my end. Fixed in [fc469dc](https://github.com/wiredtiger/wiredtiger/pull/9801/commits/fc469dc450a6b547cf1a6f653b568d792c7861fd)